src/domain/app/appSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { AppState } from "../../types";
```

Comment:
Do we really need this slice?

Reply:
not now but just keeping it maybe for future, I will comment it ....